packages/plugin-data/src/commands/force/data/bulk/upsert.ts

```diff
@@ -40,6 +40,11 @@ export default class Upsert extends DataCommand {
       description: messages.getMessage('flags.wait'),
       min: 0,
     }),
+    serial: flags.boolean({
+      char: 'r',
+      description: messages.getMessage('flags.serial'),
+      default: false,
```

Comment:
you can leave off `default:false` since that's the automatic value for booleans when they aren't there.